sample_cdr_georef/sample_cdr_georef/server.py

```diff
@@ -237,6 +237,92 @@ async def georeference_map(req: MapEventPayload,  response_model=GeoreferenceRes
     return results
 
 
+def dummy_feature_results():
+    polygon_feature_1 = {
+        "id": "1712",
+        "crs": "EPSG:4326",
+        "map_unit": 11,
+        "abbreviation": '',
+        "description": '',
+        "pattern": 'solid',
+        "color": '#dfa566',
+        "category": '',
+        "polygon_features": [
+            {
+                "features": [
+                {
+                    "id": "polygon_feature_1",
+                    "geometry": {
+                        "coordinates": [[[(-92.3368977795084, 47.96525347233585), (-92.3368977795084, 47.96520426896563), (-92.33684857613818, 47.96520426896563), (-92.33684857613818, 47.96525347233585), (-92.3368977795084, 47.96525347233585)]]]
+                    },
+                    "properties": {}
+                }
+                ]
+            },
+        ]
+    }
+
+    polygon_feature_2 = {
+        "id": "1713",
+        "crs": "EPSG:4326",
+        "map_unit": 11,
+        "abbreviation": '',
+        "description": '',
+        "pattern": 'solid',
+        "color": '#dfa566',
+        "category": '',
+        "polygon_features": [
+            {
+                "features": [
+                {
+                    "id": "polygon_feature_1",
+                    "geometry": {
+                        "coordinates": [[[(-92.01008899446106, 47.944981683802354), (-92.01008899446106, 47.94493248043213), (-92.01003979109083, 47.94493248043213), (-92.01003979109083, 47.944981683802354), (-92.01008899446106, 47.944981683802354)]]]
+                    },
+                    "properties": {}
+                }
+                ]
+            },
+        ]
+    }
+
+    feature_payload = FeatureResults(
+        cog_id="6657_9294",
```

Comment:
@Sorrento110 This should be read programmatically from the event payload - not hardcoded.  I.e., if this event is for another cog_id, the submitted result is wrong.  Also this format looks like the ngmdb_prodid_ngmdb_item_id - which is NOT the cog_id.